lms/djangoapps/django_comment_client/management/tests.py

```diff
@@ -0,0 +1,245 @@
+                "votes": {
+                    "count": 7,
+                    "down": [
+                        "3",
+                        "5",
+                        "6",
+                        "7"
+                    ],
+                    "down_count": 4,
+                    "point": -1,
+                    "up": [
+                        "1",
+                        "2",
+                        "4"
+                    ],
+                    "up_count": 3
+                }
+            }
+        cls.mongo_comment_ids.append(cls.db.contents.insert(comment_1))
+        cls.mongo_comment_ids.append(cls.db.contents.insert(comment_2))
+
+    @classmethod
+    def seed_mongo_users(cls):
+        """
+        creates test users in the users collection of the mongo database
+        """
+        user_1 = \
+            {
+                "default_sort_key": "date",
+                "notification_ids": [],
+                "external_id": "test_user_1",
+                "username": cls.sql_users[0].username
+            }
+        user_2 = \
+            {
+                "default_sort_key": "date",
+                "email": "robot2@robots.com",
+                "external_id": "test_user_2",
+                "notification_ids": [],
+                "read_states": [
+                    {
+                        "_id": "5410bb0d9fbe726175000002",
+                        "course_id": "Carnegie/Training/CLASlite",
+                        "last_read_times": {
+                            "5410bb0a9fbe727505000001": datetime.datetime.strptime('10092014', "%d%m%Y"),
+                            "541b0ef870da966ba5000001": datetime.datetime.strptime('10092014', "%d%m%Y")
+                        }
+                    }
+                ],
+                "username": cls.sql_users[1].username
+            }
+        cls.mongo_user_ids.append(cls.db.users.insert(user_1))
+        cls.mongo_user_ids.append(cls.db.users.insert(user_2))
+
+    def test_data_seed(self):
+        self.assertTrue(len(self.mongo_user_ids) == 2, 'Test mongo users were not properly seeded, rendering tests invalid.')
+        self.assertTrue(len(self.mongo_comment_ids) == 2, 'Test mongo comments were not properly seeded, rendering the tests invalid.')
+        self.assertTrue(len(self.sql_users) == 2, 'Test sql users were not properly seeded, rendering the tests invalid.')
+
+    def test_user_does_not_exist(self):
+        args = ['---@---.com', '---2@---2.com']
+        result = management_utils.rename_user(*args)
+        self.assertRegexpMatches(result['message'], 'does not exist')
+
+    def test_rename_failed(self):
+        # ensures that we throw an error if two users have the same username
+        args = [self.sql_users[0].username, self.sql_users[1].username]
+        result = management_utils.rename_user(*args)
+        self.assertRegexpMatches(result['message'], 'Error modifying the username for')
+
+    def test_change_username(self):
```

Comment:
I think much of testing in this file is superfluous. The actual
renaming, error handling, etc. is/should be tested by
`management_utils.rename_user`.

The only real logic I can see us testing here is to ensure that the
command throws a `CommandError` if an invalid number of parameters are
passed.

Checking the results of `stdout` is brittle; we want to avoid needing to
change tests each time we make small UI changes.